_health-care/_js/_components/_hello-world.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { IndexLink, Link } from 'react-router'
+
+import DateInput from './date_input';
+
+class Nav extends React.Component {
+  render() {
+    return (
+      <div>
+        <Link to="/hello">Hello</Link><br/>
+        <Link to="/night">Night</Link><br/>
+        <IndexLink to="/">Home</IndexLink>
+      </div>
+    )
+  }
+}
+
+class Hello extends React.Component {
+  render() {
+    return(
+      <div>
+        <h2>Hello World</h2>
+
+        <div className="row">
+          <h3>Happy Date</h3>
+          <DateInput date={this.props.applicationData.happyDate}
```

Comment:
Good questions...I'm struggling with this too.

I'm thinking of Hello as a panel. I think each panel should have full access to applicationData and then pass a sub-struct down to the Question. So you could imagine applicationData.personalInfo.{birthDate,name, etc} and inside the PersonalInformation panel,you'd pass `applicationData.personalInfo.birthDate` to the BirthDate question which gives you the information hiding.